### PR TITLE
changed `MANPAGER` in `README.md`

### PR DESCRIPTION
**Pull request by ronkup12**
changed MANPAGER to behave more like the default man.
- always use pager, even for small man pages
- use less with colors, in order to remove man's output when exiting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -194,7 +194,7 @@ bat main.cpp | xclip
 `MANPAGER` environment variable:
 
 ```bash
-export MANPAGER="sh -c 'col -bx | bat -l man -p'"
+export MANPAGER="sh -c 'col -bx | bat -l man -p --paging=always --color=always | less -r'"
```
Comment (SuperSandro2000) [inline README.md:197]: Do we need both `--paging=always` and `less -r`?

Comment (ronkup12) [reply README.md:197]: you are right, it does seems to work without `--paging=always`. 
Although, I do wonder if both of them are necessary when the set pager isn't less...

**Comment by sharkdp** [comment]
> use less with colors, in order to remove man's output when exiting

could you please go into a bit more detail?

**Comment by sharkdp** [comment]
I'm closing this due to inactivity. Please feel free to comment in case it should be re-opened.

**Pull request closed without merging**